frontend/src/components/AccountReceiveTableWithdrawConfirmation.vue

```diff
@@ -160,7 +160,12 @@ export default defineComponent({
     const etherscanUrl = computed(() => getEtherscanUrl(props.txHash, props.chainId)); // withdrawal tx hash URL
     const ethDisplayDecimals = 6;
     const numDecimals = isNativeToken ? ethDisplayDecimals : 2; // maximum number of decimals to show for numbers the UI (ETH fee will be small, hence the larger value)
-    const formattedAmount = round(formatUnits(amount, decimals), numDecimals); // amount being withdrawn, rounded
+    let formattedAmount: string;
+    if (isNativeToken) {
+      formattedAmount = round(formatUnits(amount, decimals), numDecimals); // amount being withdrawn, rounded
+    } else {
+      formattedAmount = roundTokenAmount(amount, props.activeFee.token);
+    }
```

Comment:
Then with the updated formatting methods we could then remove these if/else blocks to just `const formattedAmount = formatAmount(amount, tokenDecimals)`?

Reply:
Issue created [here](https://github.com/ScopeLift/umbra-protocol/issues/283)